src/controllers/softtakeover.cpp

```diff
@@ -137,5 +164,5 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
 }
 
 void SoftTakeover::ignoreNext() {
-    m_time = 0;
+    m_time = -1;
```

Comment:
document in the header file that -1 is a magic number meaning "uninitialized value"